src/frequenz/client/reporting/__main__.py

```diff
@@ -56,7 +56,9 @@ def main() -> None:
         help="End datetime in YYYY-MM-DDTHH:MM:SS format",
         required=True,
     )
-    parser.add_argument("--resolution", type=int, help="Resolution", default=None)
+    parser.add_argument(
+        "--resampling_period", type=int, help="Resampling period", default=None
```

Comment:
Is anywhere explained what unit this parameter expects? And is it desired that sub-second durations (if those are seconds) are not supported (int)?

Reply:
This is the reference to the proto file: https://github.com/frequenz-floss/frequenz-api-reporting/blob/72510043a64517dd0f7eba1cd6ea8b0fa6f898ae/proto/frequenz/api/reporting/v1/reporting.proto#L81

Reply:
Right, but when calling this program with "--help", I won't have the proto buf in mind and how something maps exactly from that to my command line parameter?

Reply:
ahhh sorry, you mean i should add something like: `help="Resampling period in seconds (integer, no sub-second durations)."` (obviously pending our discussion above).

Reply:
The parameter here is int, so that "sub second" part is not needed. But you convert it later to a time-delta which then converts it to seconds, but in floating point. And that last conversion is only the part where the problem can happen. Though rather theoretical maybe, depending what the source of the value is :)